### PR TITLE
create recipient and volunteer

### PR DESCRIPTION
**Pull request by machi1990**
##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: platform/client-admin/src/forms/create-volunteer.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const volunteerForm = new SimpleSchema({
+    firstName: {
+        type: String,
+        max: 120
+    },
+    lastName: {
+        type: String,
+        max: 120
+    },
+    email: {
```
Comment (machi1990) [inline platform/client-admin/src/forms/create-volunteer.ts:13]: is it possible with uniforms to say that this input is an email field e.g `<input type="email">`? @kingsleyzissou

Comment (wtrocki) [reply platform/client-admin/src/forms/create-volunteer.ts:13]: It is in the uniforms but not sure if that is mapped in the lib.

File: platform/client-admin/src/forms/create-volunteer.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        required: false,
+    },
+    dateOfBirth: {
+        type: Date,
+        required: false,
+        uniforms: {
+            label: "Date Of Birth"
+        },
+    },
+    canPhoneCall: {
```
Comment (wtrocki) [inline platform/client-admin/src/forms/create-volunteer.ts:45]: I thought we decided to refine this and remove?

Comment (machi1990) [reply platform/client-admin/src/forms/create-volunteer.ts:45]: Good catch. Should be removed. I forgot to remove it.

File: platform/client-admin/src/forms/create-volunteer.ts (hunk 1 above)
Comment (machi1990) [reply platform/client-admin/src/forms/create-volunteer.ts:13]: adding `uniforms: {type:'email'}` fixed it - thanks @kingsleyzissou

**Pull request merged**